test/test_http_instrumentation.py

```diff
@@ -670,7 +678,11 @@ def test_record_post_data_text_plain(self):
         encoding_type = "text/plain"
         db = self.visit('/post_request.html?encoding_type=' + encoding_type)
         post_body = self.get_post_request_body_from_db(db, True)
-        assert post_body.decode('utf8') == self.post_data_multiline_raw
+        if not isinstance(self.post_data_multiline_raw, six.text_type):
```

Comment:
This looks a bit strange. Since `self.post_data_multiline_raw` is hard-coded in the test, there should be no need to change it conditionally. Is this due to some python version incompatibilities?

Reply:
Python 2 uses byte strings as the default string type. Python 3 uses unicode. We want to convert the string to unicode (if it isn't already). I'm sure there's a proper way to do it, but adding the string with a `u` prefix was giving me errors, and this was the next thing I tried :)